data_managers/data_manager_genomad/tool_data_table_conf.xml.sample

```diff
@@ -0,0 +1,7 @@
+<tables>
+    <!-- genomad reference data  -->
+    <table name="genomad" comment_char="#">
+        <columns>value, name, path</columns>
+        <file path="tool-data/genomad.loc" />
```

Comment:
Include version column?

Reply:
The DB version number is in the `value` column 
https://github.com/Helmholtz-UFZ/galaxy-tools/blob/f5626ce54d8bab0fb01c2296b6dc85397e51dd80/data_managers/data_manager_genomad/tool-data/genomad.loc.sample#L3